domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -3281,6 +3281,45 @@ void getSQLConnection(int queryTimeout, Handler<AsyncResult<SQLConnection>> hand
     });
   }
 
+  /**
+   * Execute the given function within a transaction.
+   * <p
```

Comment:
https://google.github.io/styleguide/javaguide.html#s7.1.2-javadoc-paragraphs
suggests a different `<p>` style.

Reply:
OK. Quite mixed in RMB itself, and we're not using checkstyle plugin, FWIW. Making another attempt.